app/build.gradle.kts

```diff
@@ -8,23 +8,19 @@ plugins {
 
 android {
     namespace = "com.boolder.boolder"
-    compileSdk = 32
+    compileSdk = 33
 
     defaultConfig {
         applicationId = "com.boolder.boolder"
         minSdk = 21
-        targetSdk = 32
+        targetSdk = 33
```

Comment:
what does this do?

Reply:
This is the targeted android SDK version, or the intended version android version that the app should run on. It is usually better to keep it in sync with the `compileSdk` version.